packages/hooks/src/useP2PExchangeRate.ts

```diff
@@ -6,7 +6,7 @@ const useP2PExchangeRate = (local_currency: string) => {
 
     React.useEffect(() => {
         handleSubscription('USD', local_currency);
-    }, [handleSubscription, local_currency]);
+    }, [local_currency]);
```

Comment:
```suggestion
    // eslint-disable-next-line react-hooks/exhaustive-deps
    }, [local_currency]);
```
this is to prevent the warning.